sale_product_set/models/product_set.py

```diff
@@ -1,11 +1,20 @@
 # -*- encoding: utf-8 -*-
-from openerp import fields, models, _
+from openerp import api, fields, models, _
 
 
 class ProductSet(models.Model):
     _name = 'product.set'
     _description = 'Product set'
 
-    name = fields.Char(_('Name'), help=_('Product set name'), required=True)
+    name = fields.Char(u"Name", help=u"Product set name", required=True)
```

Comment:
unicode is not needed here, because there isn't strange chars, but anyway it doesn't harm